backstop/scenarios/tooltip.js

```diff
@@ -1,3 +1,7 @@
 const { scenario } = require('../scenarioHelper');
 
-module.exports = [scenario('basic')];
+module.exports = [
+  scenario('Type All', {
```

Comment:
I guess no need to test on hover for tooltip, but then is there a need to test all the text? Lets select only the visible tooltip one

Reply:
I think it's fine like this because selecting just the visible container would require some extra rules/changes 
![image](https://user-images.githubusercontent.com/9084735/115595448-ee394900-a2a4-11eb-8f70-1d0ce6ad5e38.png)